pkg/client/core/api/rest/certificate_rest.go

```diff
@@ -12,6 +12,12 @@ type certificateAPI struct {
 	client *HTTPClient
 }
 
+// nolint: godox
+// TODO - Consider making own endpoint to delete certificate in separate
```

Comment:
Undone TODO. Was this supposed to be a part of this cleaning? Or is it a TODO for the future

Reply:
No, this will be a new task - the calls are made to the api in the apropriate fashion like this task calls for. This TODO marks that the logic should be moved and orchestrated in the client. Trying to keep the PR in a managable state :)